### PR TITLE
Fixes for celery on Ubuntu 14.04

### PR DESCRIPTION
**Pull request by dukelion**

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -102,7 +106,7 @@ def production():
     """
     Select production instance(s)
     """
-    env.hosts = ['cabot.arachnys.com']
```
Comment (dbuxton) [inline fabfile.py]: @dukelion not sure you want this in here (although silly to have Arachnys hostname too)

Comment (dukelion) [inline fabfile.py]: it was there in master: https://github.com/arachnys/cabot/blame/master/fabfile.py#L101

Comment (dbuxton) [inline fabfile.py]: I know that cabot.arachnys.com was; I just don't think the default should change

File: Procfile [hunk 2]
```diff
@@ -1,2 +1,4 @@
 web:       gunicorn cabot.wsgi:application --config gunicorn.conf
 celery:    celery worker -B -A cabot --loglevel=INFO --concurrency=16 -Ofair
+celerybeat: python manage.py celery beat
```
Comment (dbuxton) [inline Procfile]: If there's a separate celerybeat worker we should remove the `-B` option from the celery process

**Comment by frankh** [comment]
Cleaning up old PRs

Feel free to re-open with the conflicts resolved if you think it's still relevant

**Pull request closed without merging**